nixio/property.py

```diff
@@ -97,11 +97,14 @@ def __init__(self, nixparent, h5dataset):
         self._h5dataset = self._h5group
 
     @classmethod
-    def _create_new(cls, nixparent, h5parent, name, dtype, oid=None):
+    def _create_new(cls, nixparent, h5parent, name,
+                    dtype, oid=None, shape=None):
```

Comment:
```suggestion
    def _create_new(cls, nixparent, h5parent, name,
                    dtype, shape=None, oid=None):
```

I think I prefer the shape ahead of the oid.  The forced ID should probably be always last, since it's the least useful argument.